dream-web/view/template.ml

```diff
@@ -0,0 +1,34 @@
+open Tyxml.Html
+
+let html_to_string = Fmt.to_to_string (Tyxml.Html.pp ())
+
+let instance ?(flash_messages = []) contents =
+  let flash_div (category, text) =
+    let close_x = "\u{2715}" in
+    div
+      ~a:[ a_class [ "bar"; "info" ] ]
+      [
+        div
+          ~a:[ a_class [ "close" ]; a_onclick "this.parentElement.remove()" ]
+          [ txt close_x ];
+        txt (Fmt.str "%s: %s" category text);
+      ]
+  in
+  html_to_string
+    (html
+       (head
+          (title (txt "OCaml-CI"))
+          [
+            meta ~a:[ a_charset "UTF-8" ] ();
+            link ~rel:[ `Stylesheet ] ~href:"/css/normalize.css" ();
+            link ~rel:[ `Stylesheet ] ~href:"/css/ansi.css" ();
+            link ~rel:[ `Stylesheet ] ~href:"/css/github.css" ();
+            link ~rel:[ `Stylesheet ] ~href:"/css/style.css" ();
+            link ~rel:[ `Stylesheet ] ~href:"/css/flash-messages.css" ();
```

Comment:
Can't all these files be concatenated in one file? It would avoid calling the server five times and save some bandwidth.